src/app/core/services/label.service.ts

```diff
@@ -129,26 +148,16 @@ export class LabelService {
    * @throws exception if input is an invalid color code
    */
   setLabelStyle(color: string) {
-    let r: string;
-    let g: string;
-    let b: string;
-    const white = '255';
-
-    try {
-      r = this.hexToRgb('#'.concat(color)).r.toString();
-      g = this.hexToRgb('#'.concat(color)).g.toString();
-      b = this.hexToRgb('#'.concat(color)).b.toString();
-    } catch (e) {
-      // Set rgb to white color if hexToRgb returns null
-      r = white;
-      g = white;
-      b = white;
-    }
+    let textColor: string;
+
+    textColor = (this.isDarkColor(color)) ? LIGHT_COLOR : DARK_COLOR;
```

Comment:
Minor suggestion here.
Since we are dealing with colors for labels and texts here. Let's make a clear distinction.`isDarkColor` -> `isLabelDarkColor`. IMO improves a developers understanding. @damithc what do you think?

Reply:
`isDarkColor` is a standalone function that need not be tied to labels?